Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -113,7 +108,7 @@ struct ScheduleTransferSummaryData: ScheduleTransferSectionData {
             rows.append((title: "transfer_fee_confirmation".localized(),
                          value: feeAmount.format(with: destinationCurrency)))
             rows.append((title: "transfer_net_amount_confirmation".localized(),
-                         value: destinationAmountFormattedString))
+                         value:destinationAmountFormattedString))
```

Comment:
As I mentioned in the previous comment delete destinationAmountFormattedString and use destinationAmount instead. Also remove extra (). Also do not remove space after value: